templates/taxi/car_detail.html

```diff
@@ -2,21 +2,41 @@
 
 {% block content %}
   <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
+    Model: {{ car.model }}
   </h1>
+  <br>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
   <hr>
-  <ul>
+
+  <div class="ml-3">
+    <h4>
+      Drivers
+      {% if user in car.drivers.all %}
```

Comment:
`car.drivers.all` hits db 2 times on this page. it is better to use `with` or add this queryset to the context in the corresponding view